### PR TITLE
Explore: moves add query row button below query rows

### LINKED ISSUE #15246: Explore: Query row status does not adjust when long query
The query row status (information in the top right in the input area) does not adjust when there's a long query. We should position it relative to the other content.

screenshot 2019-02-05 10 21 00

### LINKED ISSUE #20520: Explore: Move latency to QueryRowActions to get it displaying properly in long/wide explore inputs.
**What would you like to be added**:

Move latency to QueryRowActions to make sure it displays properly on wide/long inputs with overflows.

**Why is this needed**:

Makes the latency being displayed the same way no matter how long the query row input is.

### PR DESCRIPTION
**Pull request by Estrax**
**What this PR does / why we need it**:

~~Moves latency to QueryRowActions so it gets displayed the same way no matter how long the query row input is. Moves `Add query` button to the toolbar.~~

Moves latency to query row actions and `Add query` button below query rows.

**Which issue(s) this PR fixes**:

Fixes #20520 
Fixes #15246

### CODE REVIEW

**Comment by Estrax** [comment]
@jessover9000 this way?
Screenshot

**Review by davkal — CHANGES_REQUESTED** [review]
As per feedback session, we should keep per-row latencies, so perhaps simply make this PR about moving the add-query button. 

Also, the add-query button is bigger than the top toolbar buttons, I don't think that's warranted. It's not the primary or even a very prominent action. 

Update: Also noticed the responsiveness is still there in the add-query button, which is no longer needed.

**Comment by davkal** [comment]
Some small things left to do:

- Resolve conflict
- Remove overall latency (I don't think it'll be interesting)
- Make the latency field not "active" (currently it changes background on mouseover)

File: public/sass/pages/_explore.scss [hunk 1]
```diff
@@ -77,6 +77,26 @@
   }
 }
 
+.explore-container-content-item {
```
Comment (aocenas) [inline public/sass/pages/_explore.scss]: As we are trying to migrate from css to `emotion` I think we should not be creating new classes if we can avoid it.

Comment (Estrax) [reply public/sass/pages/_explore.scss]: Good catch. I'll rewrite it with `emotion` then.

**Review by davkal — COMMENTED** [review]
Is the responsiveness still needed?

**Review by davkal — APPROVED** [review]

**Pull request merged**